src/resolvers.ts

```diff
@@ -44,18 +62,27 @@ export const resolvers = {
         throw new CustomError("Amount of users must be greater than 0", 400);
       }
 
-      const users: UserPayload[] = await UserRepository.find({
+      const users: [UserPayload[], number] = await UserRepository.findAndCount({
```

Comment:
da pra dar destruct em array tb, pode ficar mais claro do que user `[0]` e `[1]`
```suggestion
      const [users, userCount]: [UserPayload[], number] = await UserRepository.findAndCount({
```